src/test/java/seedu/taassist/model/stubs/ModelStubWithNoModuleClass.java

```diff
@@ -53,6 +47,11 @@ public ObservableList<Student> getStudentList() {
         return new UniqueList<Student>().asUnmodifiableObservableList();
     }
 
+    @Override
+    public ObservableList<Student> getFilteredStudentList() {
+        return new UniqueList<Student>().asUnmodifiableObservableList();
```

Comment:
```suggestion
        return FXCollections.<Student>emptyObservableList();
```
Check if java can infer the `<Student>` argument (it should be I think).

Reply:
Nope, I tried. Java was not able to infer